docs/compass/08-01-basic-flow-tutorial.md

```diff
@@ -47,34 +47,25 @@ For simplicity reasons, use the available Order Service as the sample external A
 
-3. Create a lambda function. In the **Overview** tab, click the **Deploy new resource** button and upload the file with the [lambda function](./assets/lambda.yaml).
+3. Create a Function. In the **Overview** tab, click the **Deploy new resource** button and upload the file with the [Function](./assets/function.yaml).
 
-4. Bind your lambda:
+4. Bind your Function:
 
-    a. In the left navigation panel, go to the **Lambdas** tab and click the `call-order-service` lambda.
+    a. In the left navigation panel, go to the **Functions** tab and click the `call-order-service` Function.
 
-    b. Scroll down to the end of the Configuration view and bind your function to your instance by clicking the **Create Service Binding** button in the **Service Binding** section. Choose the ServiceInstance you want to bind your function to and click **Create Service Binding**.
+    b. Scroll down to the end of the **Configuration** view and bind your Function to your instance by clicking the **Create Service Binding** button in the **Service Binding** section. Choose the ServiceInstance you want to bind your function to and click **Create Service Binding**.
 
-    c. Save the settings in the top right corner of the page.
+5. Expose your Function:
 
-    d. Click the **Functions** tab and wait until the function status is completed and marked as `1/1`.
+    a. In the left navigation panel, go to the **API Rules** tab and click the **Create API Rule** button.
 
-5. Expose your function: 
+    b. Type `test-function` in both the **Name** and **Hostname** fields. From the **Service** dropdown, select `call-order-service`. Keep **Access strategies** as they are. Click the **Create** button in the top right corner of the page.
 
-    a. In the left navigation panel, go to the **API Rules** tab and click the **Add API Rule** button.
+    >**CAUTION:** If you use the xip.io domain with a self-signed certificate, you must edit the Gateway deployment to be able to use your Function. To do so, go to the **Deployments** tab in the left navigation panel, unfold the vertical option menu in the `{YOUR_NAMESPACE}-gateway` row, and click **Edit**. Scroll down to line 52 and set the **--skipVerify** flag to `true`. Click **Update**.
 
-    b. Type `test-function` in both the **Name** and **Hostname** fields. From the **Service** dropdown, select `call-order-service`. Keep **Access strategies** as they are.
-
-    c. Click the **Create** button in the top right corner of the page.
-
-6. Test your function. Open the terminal and run the following command:
-    ```bash
-    curl https://test-function.{CLUSTER_DOMAIN}
-    ```
-
-    This returns the following output:
+6. Test your Function. Open a separate tab in your browser and go to `https://test-function.{CLUSTER_DOMAIN}`. It will navigate you to your Function view that returns the following output:
```

Comment:
```suggestion
6. Test your Function. Open a separate tab in your browser and go to `https://test-function.{CLUSTER_DOMAIN}`. It will take you to your Function view that returns the following output:
```
`Navigate (sth)` means, more or less, to move around/through something or to find your way through something. See the [word definition in the Cambridge dictionary](https://dictionary.cambridge.org/dictionary/english/navigate).